awx/main/middleware.py

```diff
@@ -207,7 +210,7 @@ def process_request(self, request):
             request.path_info = new_path
 
 
-class MigrationRanCheckMiddleware(object):
+class MigrationRanCheckMiddleware(MiddlewareMixin):
 
     def process_request(self, request):
         executor = MigrationExecutor(connection)
```

Comment:
I would just rewrite this logic in an `__init__` method, and inherit from `object`

Reply:
In a previous commit I had the `__init__` method inherit from `object`, but by calling the mixin [here](https://github.com/django/django/blob/d3294845347a61b085e9757c757e4290979e1d10/django/utils/deprecation.py#L85) this change seems unnecessary.